Makefile

```diff
@@ -103,7 +102,8 @@ CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
 YQ ?= $(LOCALBIN)/yq
 
 ## Tool Versions
-CONTROLLER_TOOLS_VERSION ?= v0.8.0
+CONTROLLER_TOOLS_VERSION ?= v0.9.0
+YQ_VERSION ?= latest
```

Comment:
I'm curious what this change is coming along with this? Seems like it could have side effects outside of just bumping controller-tools unless they require a certain version of YQ.

Reply:
It's just an unrelated change that I snuck in here. We were hardcoding the latest version, while having variables dedicated towards the tool version, so I was just making this consistent throughout the other tooling usage.

Reply:
Sweet, no worries in that case.